recipe_scrapers/monsieurcuisine.py

```diff
@@ -1,72 +1,119 @@
 # mypy: disallow_untyped_defs=False
-from ._abstract import AbstractScraper
-from ._utils import get_minutes, get_yields, normalize_string
+import re
+
+import requests
+
+from ._abstract import HEADERS, AbstractScraper
+from ._utils import get_yields
+
+SCRIPT_PATTERN = re.compile(r'"recipeId":(\d+)')
 
 
 class MonsieurCuisine(AbstractScraper):
+    def __init__(self, url, proxies=None, timeout=None, *args, **kwargs):
+        super().__init__(url=url, proxies=proxies, timeout=timeout, *args, **kwargs)
+        scripts = self.soup.find_all("script")
+        recipe_id = None
+        for script in scripts:
```

Comment:
A suggestion since the `scripts` variable is only used for this single purpose:

```suggestion
        recipe_id = None
        for script in self.soup.find_all("script"):
```